src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOSmacOS.cs

```diff
@@ -150,11 +156,19 @@ private WKWebView OnCreateWebView(WKWebView owner, WKWebViewConfiguration config
 
 				if (!navigationArgs.Cancel)
 				{
+#if __IOS__
 					if (UIKit.UIApplication.SharedApplication.CanOpenUrl(target))
 					{
 						UIKit.UIApplication.SharedApplication.OpenUrl(target);
 						_parentWebView.OnComplete(target, isSuccessful: true, status: WebErrorStatus.Unknown);
 					}
+#else
+					if (NSWorkspace.SharedWorkspace.UrlForApplication(new NSUrl(target.AbsoluteUri)) != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['target' is null on at least one execution path.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305573)